tasklib/serializing.py

```diff
@@ -1,14 +1,13 @@
 import datetime
 import importlib
 import json
-import pytz
-import tzlocal
+from zoneinfo import ZoneInfo
 
 
 from .lazy import LazyUUIDTaskSet, LazyUUIDTask
 
 DATE_FORMAT = '%Y%m%dT%H%M%SZ'
-local_zone = pytz.timezone(str(tzlocal.get_localzone()))
+local_zone = ZoneInfo('localtime')
```

Comment:
This introduces at least one regression: the `TZ` variable is now ignored. Is that intentional?

Additionaly, there are apparently systems where `localtime` doesn't work: https://github.com/tools-life/taskwiki/issues/418 (the tasklib version string suggests it's an Arch Linux system: https://github.com/archlinux/svntogit-community/commit/89c0e644429ffcfc1a349f9d5235e09a06a97f9b#diff-37538beb61ff63edebbf735dfcf39e5d732f49183d6beb097169d971875ca422R5)
It seems a bit weird as glibc also uses `/etc/localtime`, but expecting `ZoneInfo('localtime')` certainly isn't portable.

tzlocal 3+ returns `ZoneInfo` now so perhaps reintroducing `tzlocal` here would work?

Reply:
This should be fixed in #112, can you test?

Reply:
Tested and it works on my end!

Reply:
Thanks for the discussion here guys, I was hitting the same problem and this stopped me from moving forward. Let's see if #112 indeed fixes this issue.

Reply:
It does not seem to be sufficient on my end, as I am hitting the same problem as @liskin is describing even after incorporating #112. The trace on my end looks like this:
```
>>> import zoneinfo
>>> zoneinfo.ZoneInfo("localtime")
Traceback (most recent call last):
  File "/usr/lib64/python3.11/zoneinfo/_common.py", line 12, in load_tzdata
    return resources.files(package_name).joinpath(resource_name).open("rb")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/pathlib.py", line 1044, in open
    return io.open(self, mode, buffering, encoding, errors, newline)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/tbabej/Projects/tasklib/testvenv/lib64/python3.11/site-packages/tzdata/zoneinfo/localtime'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python3.11/zoneinfo/_common.py", line 24, in load_tzdata
    raise ZoneInfoNotFoundError(f"No time zone found with key {key}")
zoneinfo._common.ZoneInfoNotFoundError: 'No time zone found with key localtime'
```

Reply:
You're not running the code from #112. There is no `zoneinfo.ZoneInfo("localtime")` anywhere at all in the codebase any more:

```
[tomi@notes tmp]$ git clone https://github.com/GothenburgBitFactory/tasklib
Cloning into 'tasklib'...
remote: Enumerating objects: 1949, done.
remote: Counting objects: 100% (272/272), done.
remote: Compressing objects: 100% (109/109), done.
remote: Total 1949 (delta 169), reused 243 (delta 163), pack-reused 1677
Receiving objects: 100% (1949/1949), 403.87 KiB | 6.73 MiB/s, done.
Resolving deltas: 100% (1179/1179), done.
[tomi@notes tmp]$ cd tasklib/
[tomi@notes (tasklib:develop) tasklib]$ gh pr checkout 112
remote: Enumerating objects: 8, done.
remote: Counting objects: 100% (8/8), done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 8 (delta 4), reused 6 (delta 4), pack-reused 0
Unpacking objects: 100% (8/8), 1.09 KiB | 558.00 KiB/s, done.
From https://github.com/GothenburgBitFactory/tasklib
 * [new ref]         refs/pull/112/head -> timezone
Switched to branch 'timezone'
[tomi@notes (tasklib:timezone) tasklib]$ git grep 'ZoneInfo.*localtime'
```

Reply:
With #112 merged `zoneinfo.ZoneInfo("localtime")` is no longer used in tasklib.

Reply:
You both are obviously correct, as I was on a wrong branch. We should rename `master` to `stable` to avoid this confusion, and make things consistent with the `taskwarrior` repo.